examples/nodejs-simple/README.md

```diff
@@ -2,11 +2,14 @@
 
 This is a very simple "server" that doesn't do much other than show how the SDK works in node.js.
 
-It will
-- Setup the Agones SDK
-- Call `sdk.ready()` to register that it is ready with Agones.
+It will:
```

Comment:
Do we want to document what the command is (or the Docker run command?) should be to run this for longer than 60 seconds?

Reply:
That is documented around line 120 after example debug output but perhaps clearer if I reorder?

Reply:
I would also like the explicit command line argument documented.

Or is it just "120" as seconds ? Should we use a key-value pair? Like --timeout=120 ?

Would be much more explicit, and also expandable down the line. WDYT?